### ISSUE TITLE
Total sent/received stats

### ISSUE THREAD

**Issue by zillionn** [ArkEcosystem/desktop-wallet#1029]
**Is your feature request related to a problem? Please describe.**
I have several wallets and I'm receiving small payments every day. Do you guys think info like this will be useful, so we'll be able to see the total amount received/sent to/from all wallets?

**Describe the solution you'd like**
Maybe something like this:

<img width="761" alt="screenshot 2019-01-27 at 12 47 46" src="https://user-images.githubusercontent.com/3462869/51800016-ec04f300-2231-11e9-9b27-924ebfa180d5.png">

**Comment by alexbarnsley** [comment]
This might be planned already. Although it's per wallet. There's a tab, either hidden or not added called "statistics". I can't remember what was to go on it based on designs, I think it was waiting on some core changes. We will revisit it at some point but will leave this issue open to discuss at the same time, as you're referring total across all wallets

**Comment by j-a-m-l** [comment]
We didn't have a real design for the statistics. When I explained the idea to Oleg, he included the price chart as an example / placeholder of some functionality.

To me, the statistics tab should be moved to cover the entire profile, so I could select only 1 wallet to focus on it or aggregating several wallets to compute the sum of their results.

Basically, I'd like to have a way, without coding, to achieve some of these examples:
 - Know how much I've received from a delegate on 1 wallet
 - Know how much I've received from from some of my delegate on all my wallets
 - Know how much I've sent to my contacts on all my wallets
 - Know how much I've sent to exchanges on all my wallets
 - Know how the balance (in ARK) of a 1 wallet evolved during the past year (with a chart)
 - Know how the balance (in EUR, USD, etc.) of a 1 wallet evolved during the past year (with a chart)
 - Know how the balance of all my wallets have evolved during this month
 - Export the rows to CSV, JSON to taxes purposes
 - These options could be named and persisted, so I could apply any of them to see the results easily.

Basically, this can be done with a form that allows selecting the wallets and filtering their transactions by several criteria.

This would be even more useful if / when we introduce tags.

So, to summarize, this would allow users to play and understand better how their wallets are evolving and use that information on the wallet and outside it.

**Comment by alexbarnsley** [comment]
@j-a-m-l we need to avoid stuff like showing rewards received from delegates, since rewards are entirely optional as per the delegate and not part of the network. A better way would be to show total sent/received for either most recent or highest amount sent/received for top 10

**Comment by j-a-m-l** [comment]
Yes, @alexbarnsley.

My examples reflect some things that users might want to do, but not the way the would do it: they could choose knowing how much they have received from any other address.

Imagine a form with fields such as:
 - From: select input that list the known wallets, wallets and contacts of the user and it can be edited to add, manually, other addresses
 - To: select input that list the known wallets, wallets and contacts of the user and it can be edited to add, manually, other addresses

In both cases we could include delegates as known addresses or not. Even including them, it's clear that we are not supporting in any manner rewards from the delegates.

**Comment by zillionn** [comment]
This sounds like a separate page `Reports`.

**Comment by j-a-m-l** [comment]
That was the original idea of the statistics, but, as you suggest, calling it "reports" is more appropriate.

**Comment by samharperpittam** [comment]
Closing this and moving it to an internal discussion for review and potential implementation.

**Comment by ArkEcosystemBot** [comment]
This issue has been closed. If you wish to re-open it please provide additional information.
